### PR TITLE
rename addAsSubmodules to useSubmodules, false by default

### PR DESCRIPTION
**Pull request by oshi97**
This commit renames the --addAsSubmodules param of jambo import and init to
--useSubmodules, and changse the default to false.

J=SLAP-1135
TEST=manual

test that I can import a theme with useSubmodules, and it will be a submodule
test jambo import --theme answers-hitchihker-theme and see it will be imported
as regular checked in files, with the a .git folder removed. check that I can commit
the theme changes without any errors/warning messages, which you would see if
you tried the .git folder wasn't removed

test importing a theme on init, but as submodule and as regular files

### CODE REVIEW

File: src/commands/import/themeimporter.js [hunk 1]
```diff
@@ -41,10 +42,9 @@ class ThemeImporter {
         description: '(deprecated: specify the themeUrl instead)'
           + ' the name of the theme to import',
       }),
-      addAsSubmodule: new ArgumentMetadata({
+      useSubmodules: new ArgumentMetadata({
         type: ArgumentType.BOOLEAN,
-        description: 'import the theme as a submodule',
-        defaultValue: true
+        description: 'import the theme as a submodule'
```
Comment (cea2aj) [inline src/commands/import/themeimporter.js:47]: Is there a reason for not setting the default here and in the describe command as false?

File: src/commands/init/initcommand.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ class InitCommand {
           + ' the name of a theme to import during the init',
         isRequired: false
       }),
-      addThemeAsSubmodule: new ArgumentMetadata({
+      useSubmodules: new ArgumentMetadata({
         type: ArgumentType.BOOLEAN, 
         description: 'if starter theme should be imported as submodule',
         defaultValue: true
```
Comment (cea2aj) [inline src/commands/init/initcommand.js:32]: defaultValue should be false

File: src/commands/init/initcommand.js [hunk 3]
```diff
@@ -48,8 +48,8 @@ class InitCommand {
           type: 'singleoption',
           options: importableThemes
         },
-        addThemeAsSubmodule: {
-          displayName: 'Add Theme as Submodule',
+        useSubmodules: {
+          displayName: 'Use Submodules',
           type: 'boolean',
           default: true
```
Comment (cea2aj) [inline src/commands/init/initcommand.js:54]: I believe this should be `default: false` as well

File: src/commands/import/themeimporter.js (hunk 1 above)
Comment (oshi97) [reply src/commands/import/themeimporter.js:47]: it looks like if we want a param to be not used by default, we've just not been including the defaultValue. Right now there's no difference between `jambo import --useSubmodules false ...` and `jambo import ...`, both functionally, and from a cbd ui perspective, but we might want to keep this distinction?

File: src/commands/init/initcommand.js (hunk 2 above)
Comment (oshi97) [reply src/commands/init/initcommand.js:32]: thanks

**Review by cea2aj — APPROVED** [review]
lgtm

**Pull request merged**